cfg/qt.cfg

```diff
@@ -5355,6 +5355,8 @@
   <define name="Q_UNUSED(X)" value="(void)(X);"/>
   <define name="QT_BEGIN_NAMESPACE" value=""/>
   <define name="QT_END_NAMESPACE" value=""/>
+  <define name="QT_BEGIN_MOC_NAMESPACE" value=""/>
```

Comment:
This looks unintentional.

Reply:
Quite intentional: https://github.com/danmar/cppcheck/actions/runs/9238225748/job/25416191350

Reply:
Ah, nice.